controllers/serving/inference_controller.go

```diff
@@ -110,7 +113,13 @@ func (ir *InferenceReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 		return ctrl.Result{}, err
 	}
 
-	// 2) Sync each predictor to deploy containers mounted with specific model.
+	// 2) Sync istio-less ingress gateway
+	if err = ir.syncIstioLessIngressGateway(&inference); err != nil {
```

Comment:
don't call "istio-less" , call it caddyIngress to be explicit, and also the docker image name

Reply:
+1

Reply:
ok